implementation/tracker.m

```diff
@@ -204,17 +211,17 @@
 
 % Initialize and allocate
 prior_weights = zeros(params.nSamples,1, 'single');
-sample_weights = cast(prior_weights, 'like', params.data_type);
+sample_weights = cast(prior_weights, class(params.data_type));
 samplesf = cell(1, 1, num_feature_blocks);
 if params.use_gpu
     % In the GPU version, the data is stored in a more normal way since we
     % dont have to use mtimesx.
     for k = 1:num_feature_blocks
-        samplesf{k} = zeros(filter_sz(k,1),(filter_sz(k,2)+1)/2,sample_dim(k),params.nSamples, 'like', params.data_type_complex);
+        samplesf{k} = complex(zeros(filter_sz(k,1),(filter_sz(k,2)+1)/2,sample_dim(k),params.nSamples))
```

Comment:
1. `zeros(N, 'like', complex_data)` will return a **complex** matrix
1. `zeros(N, class(complex_data))` will return a **real** matrix
1. `complex(zeros(N, class(complex_data)))` is what we want(same as 1/)

```
>> N=2;
>> complex_data=complex(zeros(N, 'single'));
>>
>> zeros(N, 'like', complex_data)
ans =
  2×2 single matrix
   0.0000 + 0.0000i   0.0000 + 0.0000i
   0.0000 + 0.0000i   0.0000 + 0.0000i

>> zeros(N, class(complex_data))
ans =
  2×2 single matrix
     0     0
     0     0

>> complex(zeros(N, class(complex_data)))
ans =
  2×2 single matrix
   0.0000 + 0.0000i   0.0000 + 0.0000i
   0.0000 + 0.0000i   0.0000 + 0.0000i
```